### PR TITLE
Fix minor bugs in corrdiff training

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Description

Adding missing `task` key to the hydra configs, fixing the bug in `logs` path

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by ktangsali — APPROVED** [review]
Looks good! Please merge after CI passes (needs linting)

File: examples/generative/corrdiff/conf/config_train.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ arch: ddpmpp-cwb-v0-regression
   # Network architecture [ddpmpp, ddpmpp-cwb-v2, ncsnpp, adm, ddpmpp-cwb-v0-regression]
 precond: unetregression
   # Preconditioning & loss function [vp, ve, edm, unetregression, resloss]
+task: regression
```
Comment (ktangsali) [inline examples/generative/corrdiff/conf/config_train.yaml:29]: Might be good to document the other options available here (similar to other configs)

**Review by NickGeneva — APPROVED** [review]

File: examples/generative/corrdiff/conf/config_train.yaml (hunk 1 above)
Comment (mnabian) [reply examples/generative/corrdiff/conf/config_train.yaml:29]: Done

**Pull request merged**